tokenLists/terra2.tokenlist.json

```diff
@@ -176,13 +183,20 @@
       "token": "terra1xqtstvlwcpz3kpcpw6vqh2grstw34wcezws5pawll95uuysu0hnqk6muds",
       "icon": "/tokens/vkr.jpg",
       "decimals": 6
-    }
+    },
     "terra17gck626vgax9jpe6utm7dhx4vdzawfkt0jhru03l7a3dzu98wedsfad4sz": {
       "protocol": "Depegged UST",
       "symbol": "dUST",
       "token": "terra17gck626vgax9jpe6utm7dhx4vdzawfkt0jhru03l7a3dzu98wedsfad4sz",
       "icon": "/img/dUST/dUST.png",
       "decimals": 6
+    },
+      "terra16q6kqp0p2hlfgj52sadv5p7qh2hgkkjdfske48vklz94tkxg4jwq8k8myj": {
+      "protocol": "Sayve",
+      "symbol": "SAYVE",
+      "token": "terra1xp9hrhthzddnl7j5du83gqqr4wmdjm5t0guzg9jp6jwrtpukwfjsjgy4f3",
```

Comment:
token value should match the key value. looks like a mistake here

Reply:
fixed the pisco-1 token key to match the key